### PR TITLE
feat(oui-select): add multiple selection

### PR DESCRIPTION
**Pull request by AxelPeter**
UK-65

Related to https://github.com/ovh-ux/ovh-ui-angular/pull/332

### CODE REVIEW

File: packages/oui-ui-select/ui-select.less [hunk 1]
```diff
@@ -112,67 +114,114 @@
...
+      min-height: @oui-ui-select-height;
+      line-height: @oui-ui-select-multiple-line-height;
+      padding-top: @oui-ui-select-multiple-padding-y;
+      padding-bottom: @oui-ui-select-multiple-padding-y;
+      text-align: left;
+      white-space: initial;
+
+      &:focus,
+      &_focus {
+        background-color: #f2f2f2;
```
Comment (FredericEspiau) [inline packages/oui-ui-select/ui-select.less]:
```suggestion
        background-color: @oui-color-concrete;
```

File: packages/oui-ui-select/ui-select.less [hunk 2]
```diff
@@ -112,67 +114,114 @@
...
+        border-color: #999;
```
Comment (FredericEspiau) [inline packages/oui-ui-select/ui-select.less]:
```suggestion
        border-color: @oui-color-dusty;
```

File: packages/oui-ui-select/ui-select.less [hunk 3]
```diff
@@ -112,67 +114,114 @@
...
-      opacity: 0;
-      height: 0;
-      min-height: 0;
-      padding: 0;
+    .ui-select-search {
+      width: auto !important; /* Override inline style */
       margin: 0;
-      border: 0;
+      padding: 0;
+      letter-spacing: 0.007em;
```
Comment (FredericEspiau) [inline packages/oui-ui-select/ui-select.less]: Weird, why this value ?

File: packages/oui-ui-select/ui-select.less [hunk 4]
```diff
@@ -112,67 +114,114 @@
...
+
+      max-width: 100%;
+      width: 100%;
+      height: @oui-ui-select-height;
+      text-align: left;
+
+      &:focus,
+      &_focus {
+        background-color: #f2f2f2;
+        border-color: #999;
```
Comment (FredericEspiau) [inline packages/oui-ui-select/ui-select.less]:
```suggestion
        border-color: @oui-color-dusty;
```

File: packages/oui-ui-select/ui-select.less [hunk 5]
```diff
@@ -112,67 +114,114 @@
...
+      #oui > .button-dropdown();
+
+      max-width: 100%;
+      width: 100%;
+      height: @oui-ui-select-height;
+      text-align: left;
+
+      &:focus,
+      &_focus {
+        background-color: #f2f2f2;
```
Comment (FredericEspiau) [inline packages/oui-ui-select/ui-select.less]:
```suggestion
        background-color: @oui-color-concrete;
```

File: packages/oui-ui-select/ui-select.less (hunk 1 above)
Comment (AxelPeter) [reply packages/oui-ui-select/ui-select.less]: Good catch, I'll update all in one commit.

File: packages/oui-ui-select/ui-select.less (hunk 3 above)
Comment (AxelPeter) [reply packages/oui-ui-select/ui-select.less]: It's the base letter spacing on others components. I've updated the value with the right variable.

**Pull request merged**